lib/charms/opensearch/v0/opensearch_secrets.py

```diff
@@ -78,13 +86,16 @@ def _on_secret_changed(self, event: SecretChangedEvent):
         # 3. System user hash secret update
         #     - Action: Every unit needs to update local internal_users.yml
         #     - Note: Leader is updated already
+        # 4.  S3 credentials (secret / access keys) in large relations
```

Comment:
This is covered on #282

Reply:
Right - but since this change was introduced in this PR because it was needed and was tested accordingly, I prefer that we keep it here and eventually alter it in the plugin manager PR